### PR TITLE
[SD-756] Hides search menu by selecting menu nav bar elements

### PR DESCRIPTION
**Pull request by PSzyszkaSpark**
Fixes https://spark-solutions.atlassian.net/browse/SD-756

### CODE REVIEW

File: frontend/app/assets/javascripts/spree/frontend/views/spree/layouts/spree_application.js [hunk 1]
```diff
@@ -32,4 +32,20 @@ Spree.ready(function ($) {
...
+
+  var navBarCategoryLinks = document.getElementsByClassName('main-nav-bar-category-links')
+  var navBarCategoryButtons = document.getElementsByClassName('main-nav-bar-category-button')
+  var navBarCategoryImages = document.getElementsByClassName('category-image')
+  var navbarLinks = [navBarCategoryLinks, navBarCategoryButtons, navBarCategoryImages]
+
+  $.each(navbarLinks, function(index, navbarElements) {
+    $.each(navbarElements, function(index, navBarCategoryLink) {
+      navBarCategoryLink.addEventListener('click', function () {
+        console.log(navBarCategoryLink)
```
Comment (przemosk) [inline frontend/app/assets/javascripts/spree/frontend/views/spree/layouts/spree_application.js]: do you need this `console.log` ?

Comment (PSzyszkaSpark) [reply frontend/app/assets/javascripts/spree/frontend/views/spree/layouts/spree_application.js]: It was just for test purposes and I've forgotten to remove it. Will correct it, right away.

**Review by pmilewski — CHANGES_REQUESTED** [review]
Please apply your fix to these icons as well.
Zrzut ekranu 2020-02-6 o 11 35 22

**Review by pmilewski — APPROVED** [review]
👍

**Pull request merged**